src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx

```diff
@@ -166,7 +166,7 @@ const VaPrescription = prescription => {
                     className="vads-u-margin--0 vads-u-margin-bottom--1"
                     data-testid="shipped-on"
                   >
-                    {dateFormat(shippedOn?.[i]?.completeDateTime)}
+                    {dateFormat(latestTrackingStatus?.completeDateTime)}
```

Comment:
are we supposed to be using the index here?

Reply:
The code change on line 25, `const latestTrackingStatus = prescription?.trackingList?.[0];` pulls an object, the first item out of the trackingList array. 

We [updated the vets-api serialization](https://github.com/department-of-veterans-affairs/vets-api/pull/16056) to remove the unnecessary levels of nesting, so the API will serve a 1-level array.